### PR TITLE
Pick default amount to estimate prices with based on network id

### PR DESCRIPTION
**Pull request by vkgnosis**
The previous default of 1 is problematic because it leads to bad or
failign price estimates. By having a built in default for known networks
we make running locally easier because we don't have to remember to set
this parameter.

### Test Plan
manual test

### CODE REVIEW

File: shared/src/arguments.rs [hunk 1]
```diff
@@ -120,3 +120,13 @@ pub fn wei_from_gwei(s: &str) -> anyhow::Result<f64> {
     let in_gwei: f64 = s.parse()?;
     Ok(in_gwei * 10e9)
 }
+
+pub fn default_amount_to_estimate_prices_with(network_id: &str) -> Option<U256> {
+    match network_id {
+        // Mainnet, Rinkeby
+        "1" | "4" => Some(10u128.pow(18).into()),
+        // Xdai
+        "100" => Some(10u128.pow(21).into()),
```
Comment (bh2smith) [inline shared/src/arguments.rs:129]: These amounts seem weird, you have 1*10^18 and 100*10^21 I think the 100 already make up for the +2 in the exponent.

**Review by bh2smith — APPROVED** [review]
~~Approving assuming the exponent is fixed.~~

Looks great!

File: shared/src/arguments.rs (hunk 1 above)
Comment (vkgnosis) [reply shared/src/arguments.rs:129]: It shouldn't be multiplication unless I'm missing something. The 100 is just matching on the network id which is `100` for xdai. The reason for this amount is that 10e21 should be ~ 1000 usd on xdai.

Comment (bh2smith) [reply shared/src/arguments.rs:129]: Oh ha ha! Silly me. I just read it wrong.

**Review by nlordell — APPROVED** [review]

**Pull request merged**